themes/datatogether/layouts/index.html

```diff
@@ -1,61 +1,12 @@
-                </p>
-                <!-- <a class="btn btn-large bg-red white" href="/primers">Get Started</a> -->
-            </div>
-            <div class="image col-md-4 offset-md-2"><img src="https://s3.us-east-2.amazonaws.com/static.archivers.space/add-metadata.png"></div>
-        </div>
-    </div>
-    <div class="left" style="height: 200px; width: 20%; max-width: 200px; margin-right: 25px; margin-top: 100px;"><img class="left-lines" src="https://s3.amazonaws.com/datatogether/svg/lines_left.svg"></div>
-    <div class="signup dweb-view container">
-        <div class="row">
-            <div class="image col-md-4"></div>
-            <div class="text col-md-6">
-                <p class="emphasis">
-                  This site is not a centralized data hosting service. It’s a view onto the decentralized network of individuals, organizations, projects and communities who are using the Data Together model.
-                </p>
-            </div>
-        </div>
-    </div>
-    <div class="right" style="height: 200px; width: 20%; max-width: 300px; margin-left: 25px; margin-top: 100px;"><img class="right-lines" src="https://s3.amazonaws.com/datatogether/svg/lines_right.svg"></div>
-    <div class="signup community container">
-        <div class="row">
-            <div class="text col-md-6 sub-emphasis">
-                <p>
-                  Data Together is an ongoing, inclusive conversation, an open source community, a collaboration, a model, and a collection of tools, all of which are evolving quickly. To achieve our aims, we are
-                    <!-- /react-text -->
-                    <ul>
-                      <li>Developing conceptual and technical models for communities, using decentralized and peer-to-peer infrastructure</li>
-                      <li>Articulating the patterns of actors and technologies involved, and value propositions for those actors</li>
-                      <li>Creating proof-of-concept tools that communities can use to collectively steward data</li>
-                    </ul>
-                </p>
-                <p>
-                  There are many ways to get involved and to contribute, the most important of which  is to test the Data Together model with your community and then share back where things work, where the approach opens new possibilities, and where there’s room for improvement.
-                </p>
-            </div>
-            <div class="image col-md-4 offset-md-2"></div>
-        </div>
-        <div class="container">
-          <div class="row">
-            <h2 class="tagline">Publications and Talks</h2>
-            <div class="text col-md-9">
-              <p>Code4Lib 2017 - Golden Age for Libraries - Storing Data Together</p>
-              <iframe width="560" height="315" src="https://www.youtube.com/embed/xRuPShYelm4" frameborder="0" allowfullscreen></iframe>
```

Comment:
I think we want to fix something in this iframe height/width -- I'm getting some odd column crop effects... (both simulating mobile and not) 

<img width="394" alt="screen shot 2017-08-13 at 4 00 02 pm" src="https://user-images.githubusercontent.com/4692834/29252994-a17e5b8a-8040-11e7-875b-5982d7bb3011.png">

Can we set height to 100% to resolve?